packages/react-arborist/src/components/default-container.tsx

```diff
@@ -42,38 +45,22 @@ export function DefaultContainer() {
         if (tree.isEditing) {
           return;
         }
-        if (e.key === "Backspace") {
-          if (!tree.props.onDelete) return;
-          const ids = Array.from(tree.selectedIds);
-          if (ids.length > 1) {
-            let nextFocus = tree.mostRecentNode;
-            while (nextFocus && nextFocus.isSelected) {
-              nextFocus = nextFocus.nextSibling;
-            }
-            if (!nextFocus) nextFocus = tree.lastNode;
-            tree.focus(nextFocus, { scroll: false });
-            tree.delete(Array.from(ids));
-          } else {
-            const node = tree.focusedNode;
-            if (node) {
-              const sib = node.nextSibling;
-              const parent = node.parent;
-              tree.focus(sib || parent, { scroll: false });
-              tree.delete(node);
-            }
-          }
-          return;
-        }
+
+        Object.values(shortcutHandlers).find(handler => handler.shortcut(e))?.function(tree, e);
+
+        // Focus Next Element
         if (e.key === "Tab" && !e.shiftKey) {
           e.preventDefault();
           focusNextElement(e.currentTarget);
           return;
         }
+        // Focus Previous Element
```

Comment:
I left these comments here as placeholders for the handler names. Let me know if I need to change any of these. This would turn into `handleFocusPreviousElement`.